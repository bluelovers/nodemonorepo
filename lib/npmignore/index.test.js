'use strict'
const subject = require('./index')

it('matches snapshot', () => {
  expect(subject).toMatchSnapshot()
})
