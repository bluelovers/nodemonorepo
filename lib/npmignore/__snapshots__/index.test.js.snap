// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
"
rootArray:
  - '*.log'
  - quick-test
  - test
  - /check
  - tmp
  - temp
  - tmp.*
  - temp.*
  - '*.tmp'
  - '*.temp'
  - '*.sh'
  - '*.tern-project*'
  - '*.editorconfig*'
  - '*.eslintrc*'
  - jest.config*
  - .git*
  - /.travis.yml
  - /appveyor.yml
  - /circle.yml
  - /sh
  - /coverage
  - docs
  - yarn.lock
  - package-lock.json
  - shrinkwrap*
  - renovate*
  - TODO.md
rootString: |-
  *.log
  quick-test
  test
  /check
  tmp
  temp
  tmp.*
  temp.*
  *.tmp
  *.temp
  *.sh
  *.tern-project*
  *.editorconfig*
  *.eslintrc*
  jest.config*
  .git*
  /.travis.yml
  /appveyor.yml
  /circle.yml
  /sh
  /coverage
  docs
  yarn.lock
  package-lock.json
  shrinkwrap*
  renovate*
  TODO.md

"
`;
